os/FlatLibs.mk

```diff
@@ -99,6 +99,16 @@ ifeq ($(CONFIG_ENABLE_IOTIVITY),y)
 TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)libiotivity$(LIBEXT)
 endif
 
+# Add libraries for cmsis dsp
```

Comment:
I don't want to manage this separately.
Let's add this to external library using external/Makefile.
If you have any specific reason why you want to manage this separately, please let me know

Reply:
Hi @sunghan-chang 

I'm referring the Samsung owner's GitHub repository to link the library. 
https://github.com/sahil1dotgupta/TizenRT/tree/cmsisError 

The external/cmsis directory is for storing the pre-built cmsis library, and does not contain any source files, so no Makefile is required. 
The os/*.mk files are for linking the CMSIS pre-built library.

Reply:
I don't want to modify common makefile for specific module. It will include representive modules only like app, external, framework and so on. In Tizen Lite, cmsis support is just one sub module. It's enough to add in the external library.
When we add new module, I don't want to modify other folder file. If we add Makefile in the new folder even it does not compile source codes, it will do AR to external library.

Reply:
Hi @sunghan-chang 

Removed the os/*.mk

Hello config SUCCESS to build and run the CMSIS example.

Issue:
loadable_apps failed to link the CMSIS lib: undefined symbol for CMSIS related API
Attached: build.log
[build.log](https://github.com/Samsung/TizenRT/files/15493946/build.log)


Reproduce step: PR#6071 
1. Enable CMSIS config an example
External Libraries -> [*]CMSIS NN library
Application Configuration -> Examples -> Board Specific Demos -> [*]CMSIS NN example
2. Build the image.

Please help check the external/cmsis_xx/Makefile.
Is it necessary to add EXTRA_LIBS in the os/arch/arm/scr/amebax/Make.defs.
Is there anything else we have missed to include or modify for link the lib to loadable_apps?

Thank You.